src/app/clusters/scenes/SceneTable.h

```diff
@@ -0,0 +1,363 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#pragma once
+
+#include <app-common/zap-generated/cluster-objects.h>
+#include <app/clusters/scenes/ExtensionFieldSetsImpl.h>
+#include <lib/support/CHIPMemString.h>
+#include <lib/support/CommonIterator.h>
+#include <lib/support/CommonPersistentData.h>
+#include <lib/support/Span.h>
+
+namespace chip {
+namespace scenes {
+
+enum SceneTLVTag
+{
+    kTagSceneStorageIDContainer = 1,
+    kTagSceneEndpointID,
+    kTagSceneGroupID,
+    kTagSceneID,
+    kTagSceneDataContainer,
+    kTagSceneName,
+    kTagSceneDTransitionTime,
+    kTagSceneDTransitionTime100,
+};

```

Comment:
Move to SceneTableImpl.h. This is not part of the generic interface.

Reply:
Just saw the.next comment, everything will move to implementation.